plutus-use-cases/test/Spec/Prism.hs

```diff
@@ -156,15 +156,17 @@ instance ContractModel PrismModel where
             Revoke w  -> isIssued w $~ doRevoke
             Issue w   -> isIssued w $= Issued
             Call w    -> do
-              iss  <- (== Issued)     <$> viewContractState (isIssued w)
+              iss  <- (== Issued)   <$> viewContractState (isIssued w)
               pend <- (== STOReady) <$> viewContractState (stoState w)
               when (iss && pend) $ do
                 transfer w issuer (Ada.lovelaceValueOf numTokens)
-                deposit w $ STO.coins stoData numTokens
+                let stoValue = STO.coins stoData numTokens
+                mint stoValue
```

Comment:
is this a bug?

Reply:
It was a bug. The STO tokens do get minted by this action and we had forgotten to specify it in the model.